app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,203 @@
+// Variables
+$greyLight: #f8f8f8;
+$greyMed: #ccc;
+$green: #B8F4C4;
+$red: #FF5959;
+$blue: #478BC2;
+
+// Reset
+*{
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body{
+  font-size: 16px;
+  font-family: 'Helvetica Nuee', sans-serif;
+  color: #1b1b1b;
+  line-height: 1.5;
+}
+
+// Global Navigation Styles
+.navigation{
+  position: relative;
+  z-index: 1;
+  display: flex;
+  padding: 1rem;
+  background: white;
+  color: $blue;
+  box-shadow: 1px 1px 2px rgba(black, .2);
+
+  a{
+    color: inherit;
+    text-decoration: none;
+
+    &:hover{
+      text-decoration: underline;
+    }
+  }
+
+  >div{
+    flex: 1;
+  }
+
+  .left{
+    text-align: left;
+  }
+
+  .center{
+    text-align: center;
+  }
+
+  .right{
+    text-align: right;
+  }
+
+  // Log Out link
+  .log-out{
+    border: none;
+    background: none;
+    color: $blue;
+    font-size: 1rem;
+    outline: none;
+    cursor: pointer;
+
+    &:hover{
+      text-decoration: underline;
+    }
+  }
+}
+
+
+h1{
+  font-size: 1.75rem;
+  margin-bottom: 2rem;
+}
+
+a{
+  color: $blue;
+
+	&:hover{
+		color: darken($blue, 10);
+	}
+}
+
+.container{
+  max-width: 450px;
+  margin: 2rem auto;
+  padding: 0 1rem;
+}
+
+.alert{
+  background: $red;
+  padding: .5rem;
+  text-align: center;
+  color: white;
+  margin-bottom: 1rem;
+}
+
+.notice{
+  background: $green;
+  padding: .5rem;
+  text-align: center;
+  margin-bottom: 1rem;
+}
+
+
+// List Styles
+li{
+  list-style-position: inside;
+  padding: .5rem 0;
+}
+
+dl, ul{
+	margin-bottom: 1rem;
+}
+
+dl dt{
+  font-weight: bold;
+}
+
+dl dd{
+  padding-bottom: .5rem;
+  margin-bottom: .5rem;
+  border-bottom: solid 1px #efefef;
+}
+
+// Form Styles
+form{
+  ul{
+    list-style: none;
+    li{
+      padding-bottom: .5rem;
+      margin-bottom: .5rem;
+    }
+
+  }
+  label{
+   font-size: .875rem;
+   font-weight: bold;
+   margin-bottom: .5rem;
+   display: block;
+  }
+
+  input[type="text"], input[type="password"], textarea, input[type="number"]{
+    width: 100%;
+    height: 2.5rem;
+    background: $greyLight;
+    border: solid 1px $greyMed;
+    border-radius: 2px;
+    padding: 0 1rem;
+  }
+
+  textarea{
+    padding-top: 1rem;
+  }
+}
+
+// Button Styles
+.button{
+  background: $blue;
+  color: white;
+  border: solid 1px $blue;
+  font-size: .875rem;
+  border-radius: 2px;
+  cursor: pointer;
+  text-decoration: none;
+	display: inline-block;
+	margin-right: .25rem;
+	margin-bottom: .25rem;
+	text-align: center;
+	line-height: 1rem;
+  padding: .5rem 1rem;
+	text-decoration: none;
+
+  &:hover{
+   background: darken($blue, 10);
+   color: white;
+  }
+
+  &.danger{
+		background: $red;
+    border-color: $red;
+		&:hover{
+			background: darken($red, 10);
+		}
+	}
+
+  &.grey{
+		background: $greyMed;
+    border-color: $greyMed;
+		&:hover{
```

Comment:
There are a few spots throughout the file where the indentation is off. Can you clean it up so the file looks more consistent?